.github/CODEOWNERS

```diff
@@ -4,15 +4,12 @@
 # For syntax help see:
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
-# The @googleapis/googleapis-auth is the default owner for changes in this repo
-*                       @googleapis/yoshi-java @googleapis/googleapis-auth
-
-# for handwritten libraries, keep codeowner_team in .repo-metadata.json as owner
-**/*.java               @googleapis/googleapis-auth
+# The @googleapis/cloud-java-team-teamsync is the default owner for changes in this repo
+*                        @googleapis/cloud-java-team-teamsync @googleapis/googleapis-auth
 
 
 # The java-samples-reviewers team is the default owner for samples changes
 samples/**/*.java       @googleapis/java-samples-reviewers
 
 # Generated snippets should not be owned by samples reviewers
```

Comment:
I don't think there are any generated samples in this repo, and I don't think there will be either. It is mostly a concept for GAPIC client libraries. I think we can delete this section.